pkg/test/harness.go

```diff
@@ -466,7 +466,10 @@ func (h *Harness) Setup() {
 		h.fatal(fmt.Errorf("fatal error installing crds: %v", err))
 	}
 
-	if err := testutils.WaitForCRDs(dClient, crds); err != nil {
+	if err := envtest.WaitForCRDs(h.config, crds, envtest.CRDInstallOptions{
+		PollInterval: 100 * time.Millisecond,
+		MaxTime:      10 * time.Second,
```

Comment:
I initially questioned the 10s max time... but then realized... that is what was there! :)
👏